src/main/java/org/interledger/InterledgerPacketType.java

```diff
@@ -0,0 +1,42 @@
+package org.interledger;
+
+/**
+ * An interface that defines how Interledger Packets are typed.  This interface is not an enum in
+ * order to allow this library to model non-standard packet types.
+ */
+public interface InterledgerPacketType {
+
+  Integer ILP_PAYMENT_TYPE = 1;
+  Integer ILQP_QUOTE_LIQUIDITY_REQUEST_TYPE = 2;
+  Integer ILQP_QUOTE_LIQUIDITY_RESPONSE_TYPE = 3;
+  Integer ILQP_QUOTE_BY_SOURCE_AMOUNT_REQUEST_TYPE = 4;
+  Integer ILQP_QUOTE_BY_SOURCE_AMOUNT_RESPONSE_TYPE = 5;
+  Integer ILQP_QUOTE_BY_DESTINATION_AMOUNT_REQUEST_TYPE = 6;
+  Integer ILQP_QUOTE_BY_DESTINATION_AMOUNT_RESPONSE_TYPE = 7;
+
+  /**
+   * The packet's type identifier, as specified by IL-RFC-3.
+   *
+   * @return An {@link Integer} representing the type of this packet.
+   */
+  Integer getTypeIdentifier();
```

Comment:
No major reason except using `Integer` allows the value to be `null` in Java, which for JSON deserialization purposes could be useful, whereas using an 'int' would default to 0 in Java, which would not be accurate in JSON.